workflow-bot-app/src/main/java/com/symphony/bdk/workflow/security/AuthorizationAspect.java

```diff
@@ -29,6 +30,10 @@ public void authorizationCheck(Authorized authorized) {
     String headerKey = authorized.headerTokenKey();
     HttpServletRequest httpServletRequest = getHttpServletRequest();
 
+    if (headerKey != null && httpServletRequest.getHeader(headerKey) == null) {
+      throw new UnauthorizedException(String.format(UNAUTHORIZED_EXCEPTION_MISSING_TOKEN_MESSAGE, headerKey));
```

Comment:
I would suggest to make the error message more generic for security check failing. It is a good practice, like we 'd never explicitly say in a name/pwd authentication that the username is wrong or password is wrong, but simply say credentials wrong.

Reply:
You're right, I'm now using the same error "Request token is not valid"